doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1816,6 +1819,9 @@ Response:
 
 Only the fields included in the request are updated in the subscription.
 
+Using `expires` with value equal to empty string (`""`) equals to removing the field, i.e.
```

Comment:
And same here:

    Using `expires` with a value of the empty string (`""`) equals removing the field, i.e. ...